src/components/manage/Blocks/Listing/SummaryTemplate.jsx

```diff
@@ -25,9 +25,9 @@ const SummaryTemplate = ({ items, linkTitle, linkHref, isEditMode }) => {
         {items.map((item) => (
           <div className="listing-item" key={item['@id']}>
             <ConditionalLink item={item} condition={!isEditMode}>
-              <Component componentName="PreviewImage" item={item} alt="" />
+              <Component componentName="PreviewImage" item={item} />
```

Comment:
Using PreviewImage for backwards compatibility at this time. This shoule be changed to the new Image component, but that will be a breaking change.